resources/todomvc/vanilla-examples/javascript-web-components/src/components/todo-bottombar/todo-bottombar.component.js

```diff
@@ -0,0 +1,82 @@
+import template from "./todo-bottombar.template.js";
+
+// [TO-D0]: use local package for styles when available:
+// https://github.com/WebKit/Speedometer/pull/254
+import globalStyles from "../../styles/global.constructable.js";
+import bottombarStyles from "../../styles/bottombar.constructable.js";
+
+class TodoBottombar extends HTMLElement {
+    static get observedAttributes() {
+        return ["total-items", "active-items"];
+    }
+
+    constructor() {
+        super();
+
+        const node = document.importNode(template.content, true);
+        this.element = node.querySelector(".bottombar");
+        this.clearCompletedButton = node.querySelector(".clear-completed-button");
+        this.todoStatus = node.querySelector(".todo-status");
+        this.filterLinks = node.querySelectorAll(".filter-link");
+
+        this.shadow = this.attachShadow({ mode: "open" });
+        this.htmlDirection = document.querySelector("html").getAttribute("dir") || "ltr";
+        this.shadow.host.setAttribute("dir", this.htmlDirection);
```

Comment:
I wrote in my previous comments that the same comments were applying in other files :-) can you fix those too?

1. `document.querySelector("html").getAttribute("dir")` => `document.dir`
2. `this.shadow.host` => `this`

In all components

Thanks :)